float32 confidence
uint8 class_id
float32 ssim_comp
uint16 id
